examples/hello-ruby/func.yaml

```diff
@@ -0,0 +1,5 @@
+name: hello
```

Comment:
does seem to be old-style func.yml, it has to be the one with schema and triggers.

Reply:
check our tutorials: https://github.com/fnproject/tutorials/tree/master/python/intro

Reply:
Updated func.yaml, README and loop example for triggers